### ISSUE TITLE
What is the identity class used for?

### ISSUE THREAD

**Issue by linnabrown** [cambridge-mlg/cnaps#3]
Hi authors,
Thank you very much for your great work! I am reading your code and have a question in set_enconder.py. 
https://github.com/cambridge-mlg/cnaps/blob/5a11dc0c450b0b0728de98eb64a0d1bc83035a2e/src/set_encoder.py#L42-L47

What is the class `identity` used for? Is it the `rho (.)` function in line29(see following)
https://github.com/cambridge-mlg/cnaps/blob/5a11dc0c450b0b0728de98eb64a0d1bc83035a2e/src/set_encoder.py#L29

It just seems that it does not change itself because the class `Identity` just returns the input parameter. Why you need to do post_pooling_fn step?

**Comment by jfb54** [comment]
Thanks for your interest in CNAPs. Correct. The class `Identity` is just a placeholder for the rho(.) function in the Deep Sets representation (http://papers.nips.cc/paper/6931-deep-sets.pdf). Early on, we experimented with different functions for rho(.), but in the end we just set rho(.) to identity. Thus, the `Identity`class is doing nothing and the post_pooling_fn step can be deleted. We'll do this in our next revision, but for now it has no effect.
